src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const operate = function(callback, number) {
+    this.result = callback(this.result, number);
+
+    return this;
+  };
+
+  const reset = function() {
+    this.result = 0;
+
+    return this;
+  };
+
+  const calculator = {
+    result: 0,
+    reset,
+    operate,
```

Comment:
Ok, but why to move it to constants? It's enough to just add methods to objects

Reply:
Fixed)